privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -13,20 +14,30 @@
 from art.estimators.classification import TensorFlowV2Classifier
 import string
 
+# count of shadow training sets, must be even
+AMOUNT_SETS = 2
+# ratio and size for unbalanced data sets
+SIZE_SET = 1000
+# ratios for different properties in sub-attacks
+RATIOS_FOR_ATTACK = [0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95]
 
 class PropertyInferenceAttack(Attack):
     def __init__(
-        self, target_model: Classifier, dataset: Tuple[np.ndarray, np.ndarray]
+        self, target_model: Classifier, dataset: Tuple[np.ndarray, np.ndarray], amount_sets: int = AMOUNT_SETS, size_set: int = SIZE_SET, ratios_for_attack: [int] = RATIOS_FOR_ATTACK
```

Comment:
this would take classes: [int] = CLASSES as an other input and define CLASSES= [0,1]

Reply:
Changed in new pull request so I resolved the conversation.